app.js

```diff
@@ -0,0 +1,109 @@
+// Book Class: Represents a Book
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// Store Class: Handles Storage
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(index) {
+    const books = Store.getBooks();
+
+    books.splice(index, 1);
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
+
+// UI Class: Handle UI Tasks
+class UI {
+  static displayBooks() {
+    const books = Store.getBooks();
+
+    books.forEach((book) => UI.addBookToList(book));
+  }
+
+  static addBookToList(book) {
+    const list = document.querySelector('#book-list');
+
+    const item = document.createElement('li');
+
+    item.innerHTML = `
+      <p>${book.title}</p>
+      <p>${book.author}</p>
+      <button>Remove</button>
+    `;
+
+    list.appendChild(item);
+  }
+
+  static deleteBook(el) {
+    el.remove();
+  }
+
+  static clearFields() {
+    document.querySelector('#book-title').value = '';
+    document.querySelector('#book-author').value = '';
+  }
+}
```

Comment:
- [ ] Although your app works as expected, you did not follow the instructions given in the[ project description](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md). For this milestone, you were only to work with plain objects and use javascript functions to add, remove and display books. In the [next milestone](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes.md), you are now supposed to work with classes instead of objects. Kindly work on this.

Reply:
wow that means refactoring all of our code... The instructions say that we need to use methods, which we use inside of classes. The instructions don't say that we can not use classes explicitly. What the instructions do say explicitly is not to use libraries of frameworks and we don't use that.

Reply:
Well addressed this!